### ISSUE TITLE
Symfony 3.3.8 requires a change to use environment variables in the config 

### ISSUE THREAD

**Issue by roverwolf** [qandidate-labs/qandidate-toggle-bundle#47]
Symfony 3.3.8 made a change in how it handles environment variables.

Previously (3.3.6) the following configuration worked, but no longer works in 3.3.8.
```
parameters:
    env(FEATURE_1): 'inactive'
    feature_1: '%env(FEATURE_1)%'

qandidate_toggle:
    persistence: config
    toggles:
        feature1:
            name: feature1
            status: '%feature_1%'
```

**Comment by othillo** [comment]
@roverwolf this should be fixed by https://github.com/qandidate-labs/qandidate-toggle-bundle/pull/48, can you confirm?

**Comment by roverwolf** [comment]
I can confirm that the issue is fixed in version 0.8.  Thank you.
